test/Parameters.js

```diff
@@ -74,34 +79,49 @@ describe('Parameters contract Tests', async () => {
 
   it('parameters should not be modified without admin role access', async () => {
     let tx = parameters.connect(signers[1]).setPenaltyNotRevealNum(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
```

Comment:
Hey
IMO we should have await here 
as assertRevrt is async
https://github.com/razor-network/contracts/blob/75b11fa6fb54540bc7f2a8e7fea4c4ed8eb0ff19/test/helpers/testHelpers.js#L70

I am not sure why in yesterday's call we said another way.
@SamAg19 can you elaborate on why we said so?

Reply:
Oh yes. I guess we got it confused over other assert functions in testHelpers.js.
await should be here

Reply:
Await should be here

Reply:
ok will add await @SkandaBhat @SamAg19 @abhishekvispute

Reply:
> Oh yes. I guess we got it confused over other assert functions in testHelpers.js.
> await should be here

Yes because other don't need await 
But still why are they passing without await ? @SamAg19 @abhishekvispute

Reply:
await simply forces waiting. Not having it does not guarantee failure.

Reply:
So I don't think it's necessary to add await or shall I add ? 
@SkandaBhat @SamAg19 @abhishekvispute

Reply:
its an async function. you should add await.

Reply:
yeah it's async , ok will add await